### DISCUSSION TITLE
Fear/stress affliction

### DISCUSSION THREAD

**Discussion by Hazmat616** [FakeFishGames/Barotrauma#12596] [Ideas and suggestions]
Our characters for the most part function the same regardless of what is happening be it massive flooding or watching your crew be devoured by mudraptors. So this affliction would depending on strength reduce all skills up to 50%. One complaint I've heard is how similar the crew becomes as all get high skills only having talents to set them apart. To this end each job could have a top row or 2nd row talent that protects their main skill from being affected by stress. So when faced down by monsters or giant creatures when manning a turret security will be the only ones to maintain their weapons skill. And when trying to treat yourself as you bleed to death only a doctor will not have their medical skill reduced, etc. stress would build at varying rates depending on causes and would scale with strength how much it reduces skills.

**Comment by jigardi** [comment]
10/10, would be glorious and hilarious if shouting on comms or near eachother increased stress too. xD
